core_contracts/dex/dex.py

```diff
@@ -1319,7 +1319,7 @@ def _update_total_supply_snapshot(self, _id: int) -> None:
             self._total_supply_snapshot[_id]['time'][length - 1] = current_time
 
     @external(readonly=True)
-    def balanceOfAt(self, _account: Address, _id: int, _snapshot_id: int) -> int:
+    def balanceOfAt(self, _account: Address, _id: int, _snapshot_id: int, _twa: bool = True) -> int:
```

Comment:
Code looks good, but maybe we need the default to be False for _twa?

Reply:
My thought is that since the current behavior of balanceOfAt returns a time weighted average, then the default operation of the function should be true. I'm definitely happy to change it to false though, it is pretty minor.

Reply:
I see your point after chatting some more - `False` as default is a more natural behavior. Updating in rebase